cmake/Config.cmake.in

```diff
@@ -0,0 +1,11 @@
+@PACKAGE_INIT@
+
+# Allow the usage of [PackageName]_ROOT variables for FindPackage
+if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.12")
+    cmake_policy(SET CMP0074 NEW)
+endif()
```

Comment:
As mentioned above, this requires the usage of CMake 3.7, which might not be desirable in a Config file.

Is setting the policy here required anyway? This seems to me something that should be done by the projects consuming the Config file, not by the file itself. I'd just drop this, but I'm not 100% confident so feel free to prove me wrong.